deploy/deployment/kubernetes/objects/devworkspaceroutings.controller.devfile.io.CustomResourceDefinition.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.6.1
```

Comment:
I think some generated files are missing in this PR. When I run the generate commands locally, these changes also get propagated to `deploy/bundle/manifests/controller.devfile.io_devworkspaceroutings.yaml`.

Another +1 for https://github.com/devfile/devworkspace-operator/pull/449 :)

Reply:
if you meant missing defaults, now it's fixed. Please recheck on your end.